frontend/website/src/routes/Header.svelte

```diff
@@ -34,9 +46,58 @@
 		if (isSignedIn) {
 			signedIn.set(false);
 		} else {
-			goto('/signin');
+			signin = true;
 		}
 	}
+
+	function handleLogin(event: Event): void {
+		event.preventDefault();
+		signedIn.set(true);
+	}
+
+	function validatePassword(password: string) {
+		const length = password.length >= 12;
+		const upperLower = /[a-z]/.test(password) && /[A-Z]/.test(password);
+		const lettersNumbers = /[a-zA-Z]/.test(password) && /\d/.test(password);
+		const specialChar = /[!@#?\]]/.test(password) && !/[<>]/.test(password);
```

Comment:
Fix those password validation steps, those regular expressions aren't exactly the easiest to look at or refactor if need be.